trollmoves/server.py

```diff
@@ -428,7 +428,22 @@ def __init__(self, attrs, publisher):
 
     def run(self):
         """Start listening to messages."""
-        with Subscribe('', topics=self.attrs['listen'], addr_listener=True) as sub:
+        nameserver = self.attrs.get('nameserver')
+        if nameserver in ('false', 'False'):
+            nameserver = False
+        addresses = self.attrs.get('adrresses')
+        if addresses:
+            addresses = addresses.split()
+
+        with Subscribe(
+            services=self.attrs.get('services', ''),
+            topics=self.attrs.get('topics', self.attrs['listen']),
+            addr_listener=bool(self.attrs.get('addr_listener', True)),
+            addresses=addresses,
+            timeout=int(self.attrs.get('timeout')),
+            translate=bool(self.attrs.get('translate', False)),
+            nameserver=nameserver,
+        ) as sub:
```

Comment:
Is there a test to check at least that the old behaviour is unchanged given the same config?

Reply:
Asking because codecov says that parts of this function are untested...

Reply:
Seems that the posttroll notifier (`trollmoves.server.Listener` class) has no tests. I'll see if I can figure out enough of it to create a test to see the default usage hasn't changed.

Reply:
I added the bare minimum test for this.